contracts/Staking.sol

```diff
@@ -290,7 +290,9 @@ contract Staking is Autopetrified, ERCStaking, ERCStakingHistory, IStakingLockin
 
         uint256 amount = lock.amount;
         // update total
-        account.totalLocked = account.totalLocked.sub(amount);
+        if (amount > 0) {
+            account.totalLocked = account.totalLocked.sub(amount);
```

Comment:
I think if we start following this pattern we should do it everywhere, and I think it's an overhead, you will be saving a tiny piece of gas for one case but adding one more check for all the others. Honestly, I don't think it's a good trade-off

Reply:
Makes sense. Reverting it.